rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/history/History.scala

```diff
@@ -46,8 +46,32 @@ sealed trait Trie
 sealed trait NonEmptyTrie extends Trie
 case object EmptyTrie     extends Trie
 final case class Skip(affix: ByteVector, ptr: ValuePointer) extends NonEmptyTrie {
-  def hash: Blake2b256Hash = Blake2b256Hash.create(encode.toByteVector)
-  def encode: BitVector    = codecSkip.encode(this).get
+  @SuppressWarnings(Array("org.wartremover.warts.Var"))
+  @transient
+  private[this] var _data: BitVector = null
```

Comment:
If you used `lazy val` instead of `var` with initialization in place you could get rid of the `null`s and `SuppressWarnings`